tofu/geom/utils.py

```diff
@@ -695,8 +700,40 @@ def _compute_CamLOS2D_pinhole(P=None, F=0.1, D12=0.1, N12=100,
                   'B4': 'ITER-V2',
                   'NSTX': 'NSTX-V0'}
 
-# Default config
-_DEFCONFIG = 'ITER'
+
+def _get_listconfig(dconfig=_DCONFIG, dconfig_table=_DCONFIG_TABLE,
```

Comment:
core hidden method
It is the base that provides either a dict or a str (formatted message) containing the core info

Reply:
This is great. However, I would suggest adding a short docstring to the new method and its intended use.
In particular, this would allow to describe in words what _DCONFIG and _DCONFIG_TABLE is
(to me, it's not clear what the difference is).

Reply:
Done